replay-protected-sighash.md

```diff
@@ -79,7 +80,7 @@ The 8-byte value of the amount of bitcoin spent in this input.
 * Otherwise, `hashOutputs` is a `uint256` of `0x0000......0000`.
 
 Notes:
-1. In the [original algorithm](https://en.bitcoin.it/wiki/OP_CHECKSIG), a `uint256` of `0x0000......0001` is committed if the input index for a `SINGLE` signature is greater than or equal to the number of outputs. In this BIP a `0x0000......0000` is committed, without changing the semantics.
+1. In the [original algorithm][3], a `uint256` of `0x0000......0001` is committed if the input index for a `SINGLE` signature is greater than or equal to the number of outputs. In this BIP a `0x0000......0000` is committed, without changing the semantics.
```

Comment:
nit, but not for you to resolve :  (more for @deadalnix to consider)

"In this BIP" 

This is not a BIP, although close...